lwreg/utils.py

```diff
@@ -383,13 +382,13 @@ def hash_mol(mol, escape=None, config=None):
         enable_tautomer_hash_v2=_lookupWithDefault(config,
                                                    'useTautomerHashv2'))
 
-    if _lookupWithDefault(config, 'hashConformer'):
-        escape = layers.get(RegistrationHash.HashLayer.ESCAPE, '')
-        if escape:
-            escape += '|\n'
-        layers[
-            RegistrationHash.HashLayer.ESCAPE] = escape + _get_conformer_hash(
-                mol, _lookupWithDefault(config, "numConformerDigits"))
+    # if _lookupWithDefault(config, 'hashConformer'):
+        # escape = layers.get(RegistrationHash.HashLayer.ESCAPE, '')
+        # if escape:
+            # escape += '|\n'
+        # layers[
+            # RegistrationHash.HashLayer.ESCAPE] = escape + _get_conformer_hash(
+                # mol, _lookupWithDefault(config, "numConformerDigits"))
```

Comment:
if you're going to comment out a block of code, please do it with the # symbols at the beginning of the line (or just have VS Code do it for you).

In this case please just remove the code